### PR TITLE
Created Register.jsp

### PR DESCRIPTION
**Pull request by Mariamcm90**
Created register.jsp and attached it to the registration servlet.
#9

### CODE REVIEW

File: src/test/java/codeu/controller/RegisterServletTest.java [hunk 1]
```diff
@@ -11,24 +11,25 @@
 
 public class RegisterServletTest {
 
-    private RegisterServlet registerServlet;
-    private HttpServletRequest mockRequest;
-    private PrintWriter mockPrintWriter;
-    private HttpServletResponse mockResponse;
+  private RegisterServlet registerServlet;
```
Comment (fatlotus) [inline src/test/java/codeu/controller/RegisterServletTest.java]: Did you run this one through the linter?

File: src/main/webapp/WEB-INF/view/register.jsp [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    label {
+      display: inline-block;
+      width: 100px;
+    }
+  </style>
+</head>
+<body>
+
+	<nav>
+ 	 	<a id="navTitle" href="/">CodeU Chat App</a>
```
Comment (fatlotus) [inline src/main/webapp/WEB-INF/view/register.jsp]: Nit: reindent so the tags line up

File: src/test/java/codeu/controller/RegisterServletTest.java (hunk 1 above)
Comment (Mariamcm90) [reply src/test/java/codeu/controller/RegisterServletTest.java]: I ran it through and changed it.

**Comment by fatlotus** [comment]
LGTM

**Pull request merged**